instance/templates/instance/ansible/swift.yml

```diff
@@ -0,0 +1,6 @@
+EDXAPP_SWIFT_AUTH_VERSION: '2'
+EDXAPP_SWIFT_USERNAME: '{{ user }}'
+EDXAPP_SWIFT_KEY: '{{ password }}'
+EDXAPP_SWIFT_TENANT_NAME: '{{ tenant }}'
+EDXAPP_SWIFT_AUTH_URL: '{{ auth_url }}'
+EDXAPP_SWIFT_REGION_NAME: '{{ region }}'
```

Comment:
If I do include `EDXAPP_DEFAULT_FILE_STORAGE` in the Swift config snippet, this would result in persistent databases not working until the Swift PR has been merged in edx_platform.  We could just disable Swift using the SWIFT_ENABLE switch on the instance manager for now, and see how long it takes until the Swift PR gets merged.

@antoviaque Two of the sandboxes are using persistent databases at the moment.